deployments/data-hub/data-hub-configs.yaml

```diff
@@ -38,23 +38,23 @@ data:
       objectPrefix: 'airflow-config/ejp-xml/temp-ejp-xml_files'
   europepmc.config.yaml: |-
     europePmc:
-      - dataPipelineId: europepmc_data_pipeline_biorxiv_medrxiv
+      - dataPipelineId: europepmc_data_pipeline_preprint_servers
         source:
           apiUrl: 'https://www.ebi.ac.uk/europepmc/webservices/rest/search'
           search:
-            query: '(PUBLISHER:"bioRxiv" OR PUBLISHER:"medRxiv")'
+            query: '(PUBLISHER:"bioRxiv" OR PUBLISHER:"medRxiv" OR PUBLISHER:"Research Square" OR PUBLISHER:"arXiv" OR PUBLISHER:"SSRN" OR PUBLISHER:"PsyArXiv" OR PUBLISHER:"OSF Preprints" OR PUBLISHER:"Lancet Preprints" OR PUBLISHER:"Authorea Preprints" OR PUBLISHER:"ChemRxiv")'
```

Comment:
I don't seem to get results for `Lancet Preprints` and there is only a single result for `OSF Preprints`. Are those spelled correctly?

Reply:
Yes actually I am aware of it and mentioned to Andy, he was OK to remove them but I kept incase of int the future EuropePMC will have data for them. Adding just `Lancet` is bringing some result but I am not sure it is the same server. Maybe we can add like that and just `OSF` doesn’t change the result. We can keep the like `Lancet` and `OSF` , what do you think?

Reply:
Conversation with Andy: https://github.com/elifesciences/data-hub-issues/issues/415#issuecomment-1122473407

Reply:
I see. Yes, I suspect `Lancet` and `OSF` will bring back journal article rather than preprints.
In that case maybe it is good to keep them as you suggested.
Perhaps it would be useful to add a comment to the config.